src/pages/SearchPage.js

```diff
@@ -4,11 +4,6 @@ import { FaTwitter } from "react-icons/fa";
 import "../assets/css/bookmark.css";
 import "../assets/css/search.css";
 import "../assets/css/searchPage.css";
-import {
```

Comment:
so where will the firebase config be

Reply:
so where will the firebase config be @captainPrime

Reply:
fixed